content.md

```diff
@@ -269,9 +327,60 @@ This is a simple VS Code extension that allows you to comment out your ERB files
 
 The Ruby LSP is an extension that provides performant rich features for Ruby. It connects to the ruby-lsp language server gem to analyze Ruby code and enhance the user experience
 
+## Quiz
+
+- What is the purpose of the `rbenv` tool in Ruby development?
+
```

Comment:
no newline here

Reply:
same for all questions

Reply:
Got rid of newlines after every question